e2e-product-testing/cypress/integration/Project/on-project-page.ts

```diff
@@ -0,0 +1,7 @@
+import { Given } from "cypress-cucumber-preprocessor/steps";
+
+Given(`I am on Projects page`, () => {
+  cy.visit(`https://marxan.vercel.app/projects`, {
+    timeout: 2000,
```

Comment:
I suggest to use some constant (enum?) for these timeouts, logically labeled in a way that makes it easier to choose which timeout to use - mostly to avoid having to take decisions on the exact timeout in each context.

e.g. something like

```
export enum WaitingTimesForResponses {
  subsecond = 1000,
  short = 2000,
  medium = 10000,
  long = 20000,
  giveMeABreak = 60000
};
```